packages/components/src/components/input/input.ts

```diff
@@ -686,9 +653,12 @@ export default class SdInput extends SolidElement implements SolidFormControl {
         @apply block;
       }
 
-      :host([required]) #label.has-label::after {
-        content: '*';
-        @apply ml-0.5;
+      :host([required]) #label::after {
+        content: ' *';
```

Comment:
Design explicitly states to use a space character and an asterisk, we don't need margin left in any of these cases.